test/e2e/apps/statefulset.go

```diff
@@ -52,8 +53,27 @@ const (
 
 	// Timeout for reads from databases running on stateful pods.
 	readTimeout = 60 * time.Second
+
+	// StatefulSetPoll is a poll interval for StatefulSet tests
+	StatefulSetPoll = 10 * time.Second
```

Comment:
Thanks @clarklee92 
Small nits. Others lgtm.

Do these variables have to be exported？
If it is not referenced by other packages, we can set it to private.